load_balancer.go

```diff
@@ -190,3 +144,17 @@ func (odbi *ovnDBImp) rowToLB(uuid string) (*LoadBalancer, error) {
 
 	return lb, nil
 }
+
+
+
+func (odbi *ovnDBImp) lbListImp() (*OvnCommand, error) {
```

Comment:
i think that this useless, because we already have such data in cache (or via initial sync, or after lb created via monitor)

Reply:
I added this because we dont have any function to list load_balancers.